docs/components/eyebrow/index.html

```diff
@@ -0,0 +1,104 @@
+---
+title: Eyebrow
```

Comment:
Please test the Eyebrow component in four browsers - Chrome, Firefox, IE11, Legacy Edge.

Please ask @lalithkarikelli to test for you in Safari.